src/net.cpp

```diff
@@ -2253,8 +2253,8 @@ void CConnman::SetNetworkActive(bool active)
 }
 
 CConnman::CConnman(uint64_t nSeed0In, uint64_t nSeed1In) :
-        nSeed0(nSeed0In), nSeed1(nSeed1In),
-        addrman(Params().AllowMultiplePorts())
+        addrman(Params().AllowMultiplePorts()),
+        nSeed0(nSeed0In), nSeed1(nSeed1In)
```

Comment:
Are you just re-arranging the order here?

Reply:
Yep, that's what a warning here was about (smth about "initialized earlier blablabla")

Reply:
Here is a bit more precise message :)

```
net.cpp:2256:27: warning: field 'nSeed1' will be initialized after field 'addrman' [-Wreorder]
        nSeed0(nSeed0In), nSeed1(nSeed1In),
                          ^
1 warning generated.
```